### PR TITLE
Intercom v2 documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added documentation for Intercom.
> Separate section added within the Cloud Extract category dedicated to the Airbyte-supported sources.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by RanjeetMishra — COMMENTED** [review]
LGTM!

1. Faq should highlight about version support. For intercom verson less than 2.1, syncs will error out for many intercom resources as its not supported for older versions. Customer will need to update intercom version. Check this link for version upgrade by intercom: https://developers.intercom.com/building-apps/docs/update-your-api-version. Somehow we need to highlight this as early as possible if any customer starts reading this document.

2. For above point, we should also have screen shot image for a failed sync and copying the error log which is too long. (we will work on improving the error message later). This error log as well as failed sync screenshot can be added as one section in Faq.

 Surya can share error log and screenshot for failed sync by changing API version to some old version like 1.2 or something.

**Comment by SuryaSubrahmanyam** [comment]
Full error & screenshot been mentioned here.

https://www.notion.so/rudderstacks/5ddc2843c2de41b7aa81210466e19489?v=8d5979818bbd464dabcdc618f2dfab81&p=b0bbef76379a4a8fa2fdf66bcf6aee39

File: cloud-extract-sources/airbyte-sources/intercom-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+All the Cloud Extract sources support sending data only to a data warehouse destination.
+</div>
+
+## Getting started
+
+To set up Intercom as a source on the RudderStack dashboard, follow these steps:
+
+- Log into your [**RudderStack dashboard**](https://app.rudderstack.com/).
+
+- From the left panel, select **Sources**. Then, click on **Add Source**.
```
Comment (mcchran) [inline cloud-extract-sources/airbyte-sources/intercom-v2.mdx:29]: `From the left panel` --> `At the panel on the left`

Comment (ameypv-rudder) [reply cloud-extract-sources/airbyte-sources/intercom-v2.mdx]: Thanks @mcchran for your suggestion. I've made the change.

**Comment by ameypv-rudder** [comment]
> LGTM!
> 
> 1. Faq should highlight about version support. For intercom verson less than 2.1, syncs will error out for many intercom resources as its not supported for older versions. Customer will need to update intercom version. Check this link for version upgrade by intercom: https://developers.intercom.com/building-apps/docs/update-your-api-version. Somehow we need to highlight this as early as possible if any customer starts reading this document.
> 2. For above point, we should also have screen shot image for a failed sync and copying the error log which is too long. (we will work on improving the error message later). This error log as well as failed sync screenshot can be added as one section in Faq.
> 
> Surya can share sample error log and screenshot for failed sync by changing API version to some old version like 1.2 or something.

Thanks @RanjeetMishra @SuryaSubrahmanyam for all the inputs. The suggestions have been incorporated in the document.

**Review by RanjeetMishra — APPROVED** [review]
https://github.com/rudderlabs/rudderstack-docs/blob/21bc810c10423fb7bb89d87b4c70660e00056d80/assets/cloud-extract-sources/singer-intercom-1.png this image is not showing beta flag? Is this intentional?

**Comment by ameypv-rudder** [comment]
> https://github.com/rudderlabs/rudderstack-docs/blob/21bc810c10423fb7bb89d87b4c70660e00056d80/assets/cloud-extract-sources/singer-intercom-1.png this image is not showing beta flag? Is this intentional?

This is fixed. Also added a beta stability badge on the docs.

**Pull request merged**